docs/src/equations.md

```diff
@@ -74,11 +77,11 @@ Follows the continuity equation
 
 This is discretized using the following
 ```math
-\frac{\partial}{\partial t} \rho \approx - D_h[ \rho (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))] - D^c_v[I^f(\rho \boldsymbol{u}_h)) + I^f(\rho) \boldsymbol{u}_v)]
+\frac{\partial}{\partial t} \rho \approx - D_h[ \rho (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))] - D^c_v[WI^f(J, \rho \boldsymbol{u}_h) + WI^f(J, \rho) \boldsymbol{u}_v]
```

Comment:
To understand what's going on here, we need to define what ``WI^f(J, \rho)`` is above.